web/concrete/helpers/concrete/upgrade/version_5633.php

```diff
@@ -0,0 +1,14 @@
+<?php defined('C5_EXECUTE') or die("Access Denied.");
+
+class ConcreteUpgradeVersion5633Helper {
+
+	public function run() {
+		if (!ENABLE_USER_PROFILES) {
+			$membersPage = Page::getByPath('/members');
+			if ($membersPage instanceof Page && !$membersPage->isError()) {
+				$membersPage->delete();
```

Comment:
This just caused me some work. :smile: 

We had a normal page at /about/members/ with an alias /members/ (we never used public profiles). Doing the upgrade deleted that page and all its subpages. Unfortunately the missing page was only discovered days after the upgrade, the trash had been emptied meanwhile.

Luckily I was able to restore the pages from an old backup and recreate them on the website with the <concrete5-cif> XML.

Maybe a check if it's a single page (`isGeneratedCollection`) could be in the upgrade script to avoid deletion of user created pages?